src/views/RuleEngine/Bridge/Components/BridgeConfig/BridgeHttpConfig.vue

```diff
@@ -172,7 +172,7 @@ export default defineComponent({
         pool_size: 4,
         enable_pipelining: 100,
         connect_timeout: '5s',
-        request_timeout: '5s',
+        request_timeout: '15s',
```

Comment:
We want to ensure `request_timeout` at the bridge root is the same as `request_timeout` inside `resource_opts`.  For now, making the default values match should help with quickly starting a new webhook bridge without having to fix this field.

Reply:
Ideally, we probably should show only one of these fields (the root one) and copy that value to `resource_opts.request_timeout`.